addons/search-profile/userscript.js

```diff
@@ -6,6 +6,126 @@ export default async function ({ addon, console, msg }) {
+    "aa",
+    "ao",
+    "cj",
+    "dp",
+    "dv",
+    "FB",
+    "gl",
+    "hi",
+    "jc",
+    "jm",
+    "jq",
+    "JR",
+    "k9",
+    "kc",
+    "kd",
+    "kg",
+    "LT",
+    "lu",
+    "mo",
+    "mq",
+    "nj",
+    "no",
+    "Q9",
+    "ra",
+    "rd",
+    "rr",
+    "sa",
+    "si",
+    "ta",
+    "tg",
+    "tu",
+    "tv",
+    "vh",
+    "ya",
+    "yt",
+    "gasstationwithoutpumps",
+    "programmingwithpurpose",
+    "scratchcoder123forlife",
+    "scratchteacheralpha001",
+    "scratchteacheralpha002",
+    "scratchteacheralpha003",
+    "scratchteacheralpha004",
+    "scratchteacheralpha005",
+    "scratchteacheralpha006",
+    "scratchteacheralpha007",
+    "scratchteacheralpha008",
+    "scratchteacheralpha009",
+    "scratchteacheralpha010",
+    "scratchteacheralpha011",
+    "scratchteacheralpha012",
+    "scratchteacheralpha013",
+    "scratchteacheralpha014",
+    "scratchteacheralpha015",
+    "scratchteacheralpha016",
+    "scratchteacheralpha017",
+    "scratchteacheralpha018",
+    "scratchteacheralpha019",
+    "scratchteacheralpha020",
+    "scratchteacheralpha021",
+    "scratchteacheralpha022",
+    "scratchteacheralpha023",
+    "scratchteacheralpha024",
+    "scratchteacheralpha025",
+    "scratchteacheralpha026",
+    "scratchteacheralpha027",
+    "scratchteacheralpha028",
+    "scratchteacheralpha029",
+    "scratchteacheralpha030",
+    "scratchteacheralpha031",
+    "scratchteacheralpha033",
+    "scratchteacheralpha034",
+    "scratchteacheralpha035",
+    "scratchteacheralpha036",
+    "scratchteacheralpha037",
+    "scratchteacheralpha038",
+    "scratchteacheralpha039",
+    "scratchteacheralpha040",
+    "scratchteacheralpha041",
+    "scratchteacheralpha042",
+    "scratchteacheralpha043",
+    "scratchteacheralpha044",
+    "scratchteacheralpha045",
+    "scratchteacheralpha046",
+    "scratchteacheralpha047",
+    "scratchteacheralpha048",
+    "scratchteacheralpha049",
+    "scratchteacheralpha050",
+    "scratchteacheralpha051",
+    "scratchteacheralpha052",
+    "scratchteacheralpha053",
+    "scratchteacheralpha054",
+    "scratchteacheralpha055",
+    "scratchteacheralpha056",
+    "scratchteacheralpha057",
+    "scratchteacheralpha058",
+    "scratchteacheralpha059",
+    "scratchteacheralpha060",
+    "scratchteacheralpha061",
+    "scratchteacheralpha063",
+    "scratchteacheralpha064",
+    "scratchteacheralpha065",
+    "scratchteacheralpha066",
+    "scratchteacheralpha067",
+    "scratchteacheralpha068",
+    "scratchteacheralpha069",
+    "scratchteacheralpha070",
+    "scratchteacheralpha071",
+    "scratchteacheralpha072",
+    "scratchteacheralpha073",
+    "scratchteacheralpha074",
+    "scratchteacheralpha075",
+    "scratchteacheralpha076",
+    "scratchteacheralpha077",
+    "Spongebobrectanglepants",
+    "gdpr533f604550b2f20900645890",
+    "gdpr6d746b4e0e92767d4d76ca39",
+    "gdpr86fe38473d91f98d8cf9d247",
+    "gdpr8e0f8747d5b59e360814647f",
```

Comment:
These probably shouldn't be here, they were renamed for a reason.

Reply:
> These probably shouldn't be here, they were renamed for a reason.

even though they are deleted accounts, it's visible so it'd be ok to have it their

Reply:
Yeah, we're not revealing their identities or anything. And just because there's nothing on their profiles doesn't mean we shouldn't include them. We're here to make the extension and let people use it for whatever they want, not to gatekeep what they can use it for (if that makes sense).